src/sentry/relocation/tasks/process.py

```diff
@@ -1115,6 +1170,12 @@ def _update_relocation_validation_attempt(
     retry_backoff_jitter=True,
     soft_time_limit=FAST_TIME_LIMIT,
     silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=relocation_tasks,
```

Comment:
Should we have a processing_deadline of FAST_TIME_LIMIT to align with the celery configuration?